src/app/views/events/events.jsx

```diff
@@ -94,12 +92,12 @@ const EventList = () => {
         customBodyRenderLite: (dataIndex) => {
           const item = items[dataIndex];
           return (
-            <div className="flex items-center">
-              <div className="ml-3">
+            <div className='flex items-center'>
```

Comment:
[Always use double quotes (") for JSX attributes](https://airbnb.io/javascript/react/#quotes)